### ISSUE TITLE
Starter project does not work with JS disable

### ISSUE THREAD

**Issue by lmatshoba** [bdefore/universal-redux-starter#1]
If I turn of JS on my browser nothing loads in the starter app, if I do the same with the original react readux universal everything keeps working, not sure if I am missing something?

**Comment by bdefore** [comment]
thanks for noting this @lmatshoba ... this was due to a regression recently in 2.x and has been fixed as of 2.2.4. i've updated the starter.

note that when running locally with JS off, styles will not render. this is because the webpack-isomorphic-tools does not know of the assets when in development mode. when running universal-redux-server in production (with a build made with universal-redux-build) the assets will be available and will render serverside and you should see things correctly even with JS disabled.

@halt-hammerzeit this is the workaround to the style issue locally, used in react-redux-universal-hot-example: https://github.com/erikras/react-redux-universal-hot-example/blob/master/src/helpers/Html.js#L48 ... is this still what you recommend to do? would using ExtractTextPlugin in development alleviate the issue?

**Comment by catamphetamine** [comment]
@bdefore Hello. 

> this is because the webpack-isomorphic-tools does not know of the assets when in development mode

I didn't understand what you meant there. It should know of everything everytime

> is this still what you recommend to do?

This is how I do in my project.
Your ways are not obliged to mimic mine.
If `ExtractTextPlugin` provides hot module reload (like styles do) then they're identical.

**Comment by halt-hammerzeit** [comment]
@lmatshoba 
> If I turn of JS on my browser nothing loads in the starter app, if I do the same with the original react readux universal everything keeps working, not sure if I am missing something?
No, that's a bug and it should be fixed because `isomorphic` means `with no javascript at all`, like Google bot does.

**Comment by bdefore** [comment]
@halt-hammerzeit The fault is not the webpack-isomorphic-tools but that the webpack-assets.json that's generated includes no styles in development with this webpack configuration. I'd like to explore a way to make that possible without adding too much complexity. I've created an issue for it here: https://github.com/bdefore/universal-redux/issues/37

**Comment by catamphetamine** [comment]
@bdefore oh, i see.

**Comment by marr** [comment]
Not sure if its related but the react-redux-starter-kit has a fix in place which uses `ExtractTextPlugin` to bundle css in non-dev mode, due to the `publicPath` being set to the dev server in dev:

https://github.com/davezuko/react-redux-starter-kit/blob/master/build/webpack.config.js#L214
